godotproject/Player.gd

```diff
@@ -463,7 +470,22 @@ func takeDamage(damageTaken):
 		$InvulnerableTimer.start()
 
 func die():
-	# This will need to be modified to do the death animation once that is implemented.
+	# We've died!
+	dead = true
+	# Mark us as always executing
+	pause_mode = PAUSE_MODE_PROCESS
+	# Inhibit opening the menu
+	# TODO
```

Comment:
Is this still a todo or can it be removed?

Reply:
Oops